test/aggregators/stacks-address.test.ts

```diff
@@ -1,9 +1,20 @@
 import '../setup';
 import BN from 'bn.js';
 import StacksAddress from '../../lib/aggregators/stacks-address';
+import TopBalancesAggregator, { TopBalanceAccount } from '../../lib/aggregators/top-balances';
 import TotalSupplyAggregator, { TotalSupplyResult } from '../../lib/aggregators/total-supply';
 
 
+test('get top balances', async () => {
```

Comment:
Same thing here as the other PR - let's make a separate test file for each aggregator that we test. It's just cleaner and makes it easier to only run tests for a single module.

Reply:
Done